### ISSUE TITLE
Switching between panes not working when more than 2 panes

### ISSUE THREAD

**Issue by albero94** [warpdotdev/Warp#742]
### Describe the bug

```shell
If I have more than two panes open I cannot switch into all of them using cmd+option+arrows.
```

### To Reproduce

Steps to reproduce:
2. Click on '...'
3. Scroll down to '...'
4. See error '...'

1. Split pane vertically
2. You can now switch between the left and right pane using the shortcut
3. Now split one (or both) of them horizontally
4. Using the cmd+option+arrows shortcut I'm not able to get to the 4 of them, only the horizontal arrows work but not up and down

### Expected behaviour

```shell
I would expect to move across the 4 panes.
```

### Operating System

MacOS

### OS Version

12.1

**Comment by fishinthenet** [comment]
yup, only left / right is working. Up/Down doesn't (even shortcut for up/down disappeared from the command pallete)

**Comment by elviskahoro** [comment]
@fishinthenet @albero94 hey! We accidentally lost the keybindings because we added OPT-CMD-UP/DOWN for add multicursor. See #730
Email me at elvis@warp.dev and I'll send you a link to the older release before this broke so you can use that meanwhile!

**Comment by albero94** [comment]
> #730

Thank you for the quick response! I can wait till the next release to get this fix as I'm still just exploring the terminal.
